core/shared/platform/include/platform_api_extension.h

```diff
@@ -339,6 +339,40 @@ os_socket_shutdown(bh_socket_t socket);
 int
 os_socket_inet_network(const char *cp, uint32 *out);
 
+typedef struct {
+    uint8_t addr[16];
+    uint16_t port;
+    uint8_t is_ipv4;
+    uint8_t is_tcp;
+} wamr_addr_info_t;
```

Comment:
How about changing to bh_addr_info_t or bh_sock_addr_info_t? We define type bh_socket_t in platform layer, had better align to it.

Reply:
sure, didn't know about the convention. What does 'BH' stand for?

Reply:
bh is short for Beihai, which is a code name of internal project, some of WAMR core/share APIs were ported from it.

Reply:
Thanks, we might want to do a rename at one point of time to avoid internal code names and make it more clear for people outside your organization (that will probably improve the onboarding experience), but let's keep it like that for now.